auslib/db.py

```diff
@@ -1828,7 +1828,7 @@ def getReleases(self, name=None, product=None, limit=None, transaction=None):
         rows = self.select(columns=[self.name, self.product, self.data_version],
                            where=where, limit=limit, transaction=transaction)
         for row in rows:
-            row["data"] = self.getReleaseBlob(row["name"], transaction)
+            row["data"] = next(iter(self.getReleaseBlobs(names=[row["name"]], transaction=transaction).values()))
```

Comment:
I see this `next(iter(...))` pattern being used a lot where you're replacing `getReleaseBlob` calls. Perhaps it's worth keeping `getReleaseBlob` as a thin wrapper for `getReleaseBlobs` that maintains the old interface?